examples/Producer/Program.cs

```diff
@@ -104,10 +90,22 @@ public static void Main(string[] args)
                         val = text.Substring(index + 1);
                     }
 
-                    var deliveryReport = producer.ProduceAsync(topicName, key, val);
-                    var result = deliveryReport.Result; // synchronously waits for message to be produced.
-                    Console.WriteLine($"Partition: {result.Partition}, Offset: {result.Offset}");
+                    // calling .Result on the asynchronous produce request below causes it to
```

Comment:
Capital c in the calling

Reply:
i thought all the kool kids started their sentences in lower case these days. But I get your point, C# isn't cool enough.